examples/clock/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "clock"
+version = "0.1.0"
+authors = ["Richard Dodd <richard.o.dodd@gmail.com>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+gloo = { path = "../..", features = ["futures"] }
+wasm-bindgen = "0.2.54"
+wasm-bindgen-futures = "0.4.4"
+futures-util = "0.3"
+chrono = { version = "0.4.10", features = ["wasmbind"] }
+console_error_panic_hook = "0.1.6"
+
+[dependencies.web-sys]
+version = "0.3.19"
+features = [
+    "console",
+    "Window",
+    "Document",
+    "Element",
+    "Node",
+]
+
+[workspace]
```

Comment:
Is there a reason for this `[workspace]`?

Reply:
It's here so that the root `Cargo.toml` doesn't complain about it not being in the root workspace. (try removing it and building to see the problem).

Reply:
Ah, okay.